vaadin-notification-flow-demo/src/main/java/com/vaadin/flow/component/notification/demo/NotificationView.java

```diff
@@ -141,4 +146,129 @@ private void addStyledNotificationContent() {
         button.setId("styled-content-notification-button");
         addCard("Notification with styled content", button);
     }
+
+    private void createThemeVariants() {
+        createDefault();
+        createPrimary();
+        createContrast();
+        createSuccess();
+        createError();
+    }
+
+    private void createDefault() {
+        // begin-source-example
+        // source-example-heading: Default
+        Notification notification = new Notification(
+                new Text("Please update your password"));
+
+        Button notNotButton = new Button("Not now", e -> notification.close());
```

Comment:
Nit: should some spacing be added between the buttons? Like on webcomponent demo page.

Reply:
Done.